lib/l10n/intl_en.arb

```diff
@@ -9,6 +9,7 @@
   "labelEmail": "Email",
   "labelPassword": "Password",
   "labelConfirmPassword": "Confirm password",
+  "labelAskPermissions": "Ask permissions!",
```

Comment:
Avoid exclamation marks: https://uxplanet.org/effective-writing-for-your-ui-things-to-avoid-f6084e94e009
```suggestion
  "labelAskPermissions": "Request editing permissions",
```